src/analysis/schwartz_space.lean

```diff
@@ -392,6 +395,41 @@ instance : uniform_add_group 𝓢(E, F) :=
 instance : locally_convex_space ℝ 𝓢(E, F) :=
 seminorm_family.to_locally_convex_space (schwartz_with_seminorms ℝ E F)
 
+instance : topological_space.first_countable_topology (𝓢(E, F)) :=
+(schwartz_with_seminorms ℝ E F).first_countable
+
 end topology
 
+section distribution
+
+variables (𝕜 F)
+variables [is_R_or_C 𝕜] [normed_space 𝕜 F] [smul_comm_class ℝ 𝕜 F]
+
+/-- The delta distribution as a linear map. -/
+def delta_aux (x₀ : E) : 𝓢(E, F) →ₗ[𝕜] F :=
+{ to_fun := λ f, f x₀,
+  map_add' := λ f g, add_apply,
+  map_smul' := λ a f, smul_apply }
+
+lemma delta_aux_apply (x₀ : E) (f : 𝓢(E, F)) : delta_aux 𝕜 F x₀ f = f x₀ := rfl
+
+/-- The delta distribution -/
+def delta (x₀ : E) : 𝓢(E, F) →L[𝕜] F :=
```

Comment:
Can you use this to provide a `t2_space` instance?

Reply:
I guess, but I would like to wait with providing the `t2_space` and `complete_space` instances until they are actually needed.